### PR TITLE
Added shared-task BEA 2024

### PR DESCRIPTION
**Pull request by KaiNorth**
Hi Anaïs,

I have added a single markdown for our BEA shared-task, named 2024.md. 

Could you double check that everything is okay and help us with the urls so that it displays on the website?

Thanks again for your help with this!

Kai

### CODE REVIEW

File: _pages/sharedtask/2024.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+- **Item difficulty**: A measure of item difficulty where higher values indicate a higher proportion of examinees answered the item correctly (less difficult items).
+
+- **Time intensity**: arithmetic mean response time, measured in seconds, across all examinees who attempted a given item in a live exam. This includes all time spent on the item from the moment it is presented on the screen until the examinee moves to the next item, as well as any revisits.
+
+
+The data is structured as follows:
+
+| ItemNum | ItemStem_Text | Answer__A | Answer__B| Answer__C| Answer__D| Answer__E| (…) | Answer_Text | PVALUE | Response_Time | EXAM |
+|---|---|---|---|---|---|---|---|---|---|---|---|
```
Comment (anaistack) [inline _pages/sharedtask/2024.md:55]: There is a problem with this table format. The table is not correctly visualized.

**Comment by KaiNorth** [comment]
Hi Anaïs, I've removed the table, and have updated _posts and navigation.yml. Let me know if anything else needs fixing. Thanks for this!

File: _pages/sharedtask/2024.md (hunk 1 above)
Comment (KaiNorth) [reply _pages/sharedtask/2024.md]: Should be resolved now :)

**Comment by anaistack** [comment]
Looking good! I'm closing this PR.

**Review by anaistack — APPROVED** [review]

**Pull request merged**